ground/src/main/java/com/google/android/ground/ui/home/HomeScreenFragment.kt

```diff
@@ -147,9 +159,62 @@ class HomeScreenFragment :
       R.id.sync_status -> homeScreenViewModel.showSyncStatus()
       R.id.nav_offline_areas -> homeScreenViewModel.showOfflineAreas()
       R.id.nav_settings -> homeScreenViewModel.showSettings()
-      R.id.nav_sign_out -> homeScreenViewModel.showSignOutConfirmation()
+      R.id.nav_sign_out -> showSignOutConfirmationDialog()
     }
     closeDrawer()
     return true
   }
+
```

Comment:
Is there a way to move this and other side panel logic into their own files? Ok to do in a follow up.

Reply:
Moved the compose code to a separate file.